src/components/action-button.js

```diff
@@ -10,17 +10,20 @@ export const runActions = async actionList => {
 };
 
 export default function renderButton({ layout, Theme, app, context, senseNavigation, element }) {
+  const isSense = !!senseNavigation;
```

Comment:
Feels like we are checking if we are in sense somewhere else? Anyhow, the navigation api always exists inside sense right?

Reply:
I think we do, but also based on navigation I think. So this is fine